code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -485,6 +487,13 @@ Difficulty: Hard
 	burst_range = initial(burst_range) + round(anger_modifier * 0.08)
 	beam_range = initial(beam_range) + round(anger_modifier * 0.12)
 
+/mob/living/simple_animal/hostile/megafauna/hierophant/bullet_act(obj/item/projectile/P)
+	if(stat == CONSCIOUS && !target && AIStatus != AI_OFF && !client)
+		if(P.firer && get_dist(src, P.firer) <= aggro_vision_range)
+			FindTarget(list(P.firer), 1)
+		Goto(P.starting, move_to_delay, 3)
+	..()
+
```

Comment:
did you mean to add this block in this PR?

If so could you document the change in the PR description

Reply:
Yup, sure. What this does is make the hierophant move towards any bulllet that hits them, in this case colossus bolts. If colossus is in the arena when the hierophant is alive and shooting at someone, hierophant will aggro quite fast.